src/app/components/Form/index.tsx

```diff
@@ -27,14 +38,18 @@ const CurrencyInput = styled(Box)`
   border: 2px solid #0c2a4d;
   background-color: #0c2a4d;
   color: #ffffff;
-  padding: 3px 20px;
+  padding: 3px 7px;
   height: 40px;
   border-radius: 10px;
   outline: none;
   width: 100%;
   -webkit-appearance: none;
   transition: border 0.3s ease;
   overflow: visible;
+
+  @media screen and (min-width: ${smallScreenSize}) {
```

Comment:
The current breakpoints in the theme file don't fit the different devices? @hetfly 
If it is not fit, then I'd like you to define the breakpoints and use them.

Reply:
updated, thanks!